README.md

```diff
@@ -6,25 +6,118 @@ Metrilo helps you track the events that the people visiting your shop do - viewi
-2. [Import your resources](#importing-resources) to Metrilo
-3. Send [resource updates](#creating-and-updating-resources) whenever your resources change
-4. [Install the tracking library](#tracking-library-installation) on each page of your website that you want to track
-5. Send [tracking events](#sending-tracking-events) to Metrilo for each user action on your website
+### Quickstart guide
 
-That's it! Now user actions on your website will be recorded in your [Metrilo project](https://app.metrilo.com).
+:warning: If you decide to follow along, you'll end up with some resources that you (most probably) won't use in the future. However, that's perfectly fine - after you're done testing, you can just ping us and we'll reset your project to a pristine state.
 
-## API
+Once you've [signed up for a Metrilo account](https://app.metrilo.com/signup), navigate to the `Settings -> Installation` page in your [Metrilo project](https://app.metrilo.com) and keep your `API Token` handy - you'll need it for the rest of this tutorial:
+
+![](images/api-token.png)
+
+First, we need to create some resources. Let's create a category, a product and a customer. We'll assume your API token is `sh0p-t0k3n` and the current time (in milliseconds since 1970) is `1518004715732`:
+
+```
+curl -X POST https://trk.mtrl.me/category" -H "Content-Type: text/plain" -d \
+'{
+  "time": 1518004715732,
+  "token": "sh0p-t0k3n",
+  "params": {
+    "id": "1",
+    "name": "Awesome clothing",
+    "url": "https://my-awesome-shop.com/awesome-clothing"
+  }
+}'
+```
+
+```
+curl -X POST https://trk.mtrl.me/product" -H "Content-Type: text/plain" -d \
+'{
+  "time": 1518004715732,
+  "token": "sh0p-t0k3n",
+  "params": {
+    "id": "2",
+    "categories": ["1"],
+    "name": "Awesome T-Shirt",
+    "url": "https://my-awesome-shop.com/awesome-clothing/awesome-t-shirt"
+  }
+}'
+```
+
+```
+curl -X POST https://trk.mtrl.me/customer" -H "Content-Type: text/plain" -d \
+'{
+  "time": 1518004715732,
+  "token": "sh0p-t0k3n",
+  "params": {
+    "email": "johnybravo@gmail.com",
+    "createdAt": 1518004715732
+  }
+}'
+```
+
+The next step is to install the frontend library. Add the following snippet to your website (preferably the `<head>` tag):
+
+```
+<script type="text/javascript" src="https://trk.mtrl.me/tracking.js?token=sh0p-t0k3n"></script>
+```
+
+When you load that script, you should have access to a `window.metrilo` object. That's what you're going to use to create tracking events on behalf of your customers.
+
+Now let's create the following user journey - the user opened your homepage, navigated to the category page, then the product, then added it to the cart and finally placed an order. Here is what you would do on the frontend:
```

Comment:
I think displaying the user journey with bullet points would be better - visually separating the journey steps on separate rows will make the associations to the specific library calls more obvious.

Reply:
Well, the `placing order` part is not exactly one point - that's why I did it in one sentence. I'll think a bit more and try to make it clearer